app/scripts/lib/setupSentry.js

```diff
@@ -461,6 +46,18 @@ export default function setupSentry() {
 
   log('Initializing');
 
+  // Normally this would be awaited, but getSelf should be available by the time the report is finalized.
+  // If it's not, we still get the extensionId, and we'll just log an error for installType.
```

Comment:
Here's a suggestion to make this align better with how this seems to behave.

I know that we do log an error if `getSelf` fails, but this comment seems to suggest that we'd log an error if the `installType` wasn't available yet. e.g. if the error was reported before `getSelf` resolved. Really in that case there would be no error.
```suggestion
  // If it's not, we still get the extensionId, but the installType will default to "unknown"
```

Reply:
Yeah, that is a lot clearer, I agree. Thanks!